doc/developer-guides/hld/ivshmem-hld.rst

```diff
@@ -247,7 +247,7 @@ architecture and threat model for your application.
 
    - The previously highlighted technologies rely on the kernel, as a secure component, to enforce such policies. Because of this, we strongly recommend enabling secure boot for the Service VM, and extend the secureboot chain to any post-launched VM kernels.
    - To ensure no malicious software is introduced or persists, utilize the filesystem (FS) verification methods on every boot to extend the secure boot chain for post-launch VMs (kernel/FS).
-   - Reference: ACRN secure boot extension guide (`ClearLinux <https://projectacrn.github.io/latest/tutorials/enable_laag_secure_boot.html?highlight=secure%20boot>`_, `Windows <https://projectacrn.github.io/latest/tutorials/waag-secure-boot.html>`_)
+   - Reference: ACRN secure boot extension guide (`Windows <https://projectacrn.github.io/latest/tutorials/waag-secure-boot.html>`_)
```

Comment:
How about same this? Then it's of same style with the following `dm-verity` reference.

 -   - Reference: ACRN secure boot extension guide (`Windows <https://projectacrn.github.io/latest/tutorials/waag-secure-boot.html>`_)
+   - Reference:  `Enable Secure Boot in Windows <https://projectacrn.github.io/latest/tutorials/waag-secure-boot.html>`_
    - Reference Stack:  `dm-verity <https://www.kernel.org/doc/html/latest/admin-guide/device-mapper/verity.html>`_

Reply:
> How about same this? Then it's of same style with the following `dm-verity` reference.
> 
> Reference: `Enable Secure Boot in Windows <https://projectacrn.github.io/latest/tutorials/waag-secure-boot.html>`_
> Reference Stack: `dm-verity <https://www.kernel.org/doc/html/latest/admin-guide/device-mapper/verity.html>`_

This `:ref:` function only works for documents in ReST that are part of our documentation. I don't believe we have any better option for `dm-verity` (unless we can also cross-reference with ReST in the `acrn-kernel` repo, @dbkinder ?)

Reply:
For references to ACRN documents (on projectacrn.github.io) we should use ``:ref:`doc_title` `` instead of hard coding links to ``https://projectacrn.github.io/lastest/...``.  If you use the ``https://...`` link that would link to the wrong document when we make the release-specific documentation version (e.g., for the 2.1 release).  For references to documentation to other sites, such as on the  kernel.org site, we don't have a choice and need to link to their documentation using ``https://..``